x-pack/plugins/osquery/public/scheduled_query_groups/form/index.tsx

```diff
@@ -281,8 +281,12 @@ const ScheduledQueryGroupFormComponent: React.FC<ScheduledQueryGroupFormProps> =
   }, [agentPoliciesById, policyId]);
 
   const handleNameChange = useCallback(
-    (newName: string) => setFieldValue('name', newName),
-    [setFieldValue]
+    (newName: string) => {
+      if (getFields().name.value === '') {
```

Comment:
current `name` field value should be taken from `useFormData()` https://github.com/elastic/kibana/pull/112381/files#diff-a09490f9ec7d029e9db73e264517b3cf4f4628f642440d4eca50bb54cd75d79eL263